core/src/main/java/com/sequenceiq/cloudbreak/controller/validation/template/TemplateValidator.java

```diff
@@ -80,7 +80,7 @@ public void validate(Credential credential, InstanceGroup instanceGroup, Stack s
                 }
                 if (vmType == null) {
                     validationBuilder.error(
-                            String.format("The '%s' instance type isn't supported by '%s' platform",
+                            String.format("Our platform currently not supporting the '%s' instance type for '%s'",
```

Comment:
can we somehow highlight what are supported?

Reply:
unfortunately not, or it wouldn't be nice at least since, for example, we're supporting 60 different instance types for the eu-central-1a location. other locations may have more so the result message would be enormous.